### PR TITLE
feature: implement mention link

### PR DESCRIPTION
**Pull request by johnanisere**
### Linked issue
User description's clickable GitHub mentions. As mentioned in issue #59.

### CODE REVIEW

**Comment by johnanisere** [comment]
@byCedric Have you seen this? what do you think?

File: src/molecules/user/user.js [hunk 1]
```diff
@@ -1,19 +1,31 @@
...
 
 const decorators = {
-	'@': (segment, match, key) => <UserDescriptionHighlight key={key}>{segment}</UserDescriptionHighlight>,
-	'#': (segment, match, key) => <UserDescriptionHighlight key={key}>{match}</UserDescriptionHighlight>,
+	"@": (segment, match, key) => (
+		<UserDescriptionHighlight
+			key={key}
+			href={mentionLink(segment)}
+			target="_blank"
+			rel="noopener noreferrer"
```
Comment (byCedric) [inline src/molecules/user/user.js]: I always forget the `rel` attributes on these things 😄 Thanks! I think the `noopener` is a safe one, but I'm not sure about the `noreferrer`. Do you have any thoughts in this one?

File: src/molecules/user/user.js [hunk 2]
```diff
@@ -1,19 +1,31 @@
...
+	UserDescriptionHighlight
+} from "./elements";
 
 const decorators = {
-	'@': (segment, match, key) => <UserDescriptionHighlight key={key}>{segment}</UserDescriptionHighlight>,
-	'#': (segment, match, key) => <UserDescriptionHighlight key={key}>{match}</UserDescriptionHighlight>,
+	"@": (segment, match, key) => (
+		<UserDescriptionHighlight
+			key={key}
+			href={mentionLink(segment)}
```
Comment (byCedric) [inline src/molecules/user/user.js]: I must add a bit more documentation to the highlighter atom I think. In the `mentionLink` you separate the special character `@` from the text right? Luckily I thought about this use case when I built the highlighter, it's the `match` parameter 😄  But again, this is fully my fault for lack of documentation.

---

TL;DR; The `match` parameter is the value you can use here, without having to separate the character from text yourself 😄 

---

The decorator is invoked with the segment, match and key. Take this text for example:

```
Something something @abc somehting @qwe
```

This will result in 2x method invocations with the `@` decorator with the following arguments:

invocation | segment | match | key
---------- | ------- | ----- | ---
\# 0       | `@abc`  | `abc` | `0`
\# 1       | `@qwe`  | `qwe` | `1`

**Comment by byCedric** [comment]
Hi @johnanisere! I like it, thanks! I'll take care of some small code styling adjustments. I've also added 2 comments, it's nothing big 😄 Thanks a lot ❤️ 

FYI, I might move the highlight decorators to the `organisms/github-user` component. This way we have a better separation of concerns and no references to github whatsoever in `moluecules/user` component. 😄
